### PR TITLE
translator level up

### PR DESCRIPTION
**Pull request by asasin098676**

### CODE REVIEW

File: src/assets/i18n/en.json [hunk 1]
```diff
@@ -18,6 +18,34 @@
     "ANALYTICS_FALLBACK": "Some analytics here in the future",
     "SIDENAV": {
       "SLOTS": "SLOTS"
+    },
+    "CREATE_TIME_SLOT_CARD": {
```
Comment (aleks00onyshko) [inline src/assets/i18n/en.json]: remove `_CARD`

File: src/assets/i18n/en.json [hunk 2]
```diff
@@ -18,6 +18,34 @@
...
+      "START_TIME": "Start Time",
+      "DURATION": "Duration",
+      "ERROR": "Slot will end in the next day"
+    },
+    "TIME_SLOT":{
+      "LESSON": "Piano",
```
Comment (aleks00onyshko) [inline src/assets/i18n/en.json]: LESSON_TYPE

File: src/assets/i18n/en.json [hunk 3]
```diff
@@ -18,6 +18,34 @@
...
+      "TEACHER": "Maria Boyko",
```
Comment (aleks00onyshko) [inline src/assets/i18n/en.json]: remove this

File: src/assets/i18n/en.json [hunk 4]
```diff
@@ -18,6 +18,34 @@
...
+      "MIN": "min"
+    },
+    "TIME_SLOT_LIST":{
+      "NO_SLOT": "No slots for this day"
```
Comment (aleks00onyshko) [inline src/assets/i18n/en.json]: NO_SLOTS

File: src/assets/i18n/en.json [hunk 5]
```diff
@@ -18,6 +18,34 @@
...
+    },
+    "CREATE_TIME_SLOT_DIALOG":{
+      "TITLE": "Create new lesson",
+      "CANCEL": "Cancel",
```
Comment (aleks00onyshko) [inline src/assets/i18n/en.json]: Move to COMMON

File: src/assets/i18n/uk.json [hunk 6]
```diff
@@ -19,6 +19,33 @@
...
       "SLOTS": "Часові Слоти"
+    },
+    "CREATE_TIME_SLOT_CARD": {
+      "START_TIME": "Початковий час",
+      "DURATION": "Тривалість",
+      "ERROR": "Слот закінчиться наступного дня"
+    },
+    "TIME_SLOT": {
+      "LESSON": "Фортепіано",
+      "TEACHER": "Марія Бойко",
```
Comment (aleks00onyshko) [inline src/assets/i18n/uk.json]: remove

File: src/assets/i18n/uk.json [hunk 7]
```diff
@@ -19,6 +19,33 @@
...
+      "MIN": "хв"
+    }, 
+    "TIME_SLOT_LIST":{
+      "NO_SLOT": "Не має слотів на цей день"
+    },
+    "CREATE_TIME_SLOT_DIALOG": {
+      "TITLE": "Створіть новий урок",
+      "CANCEL": "Закрити",
```
Comment (aleks00onyshko) [inline src/assets/i18n/uk.json]: Don't forget

**Review by aleks00onyshko — CHANGES_REQUESTED** [review]

File: src/assets/i18n/en.json (hunk 1 above)
Comment (aleks00onyshko) [reply src/assets/i18n/en.json:22]: fixed in https://github.com/aleks00onyshko/touche-adminstration-FE/pull/6/commits/65576347a418f94dd3396e1348ffce56e9fea0f1

Comment (asasin098676) [reply src/assets/i18n/en.json]: Resolved in 71246f23aebad5a25b7068cd1cd89704b5960a38

File: src/assets/i18n/en.json (hunk 2 above)
Comment (asasin098676) [reply src/assets/i18n/en.json]: Resolved in 71246f23aebad5a25b7068cd1cd89704b5960a38

File: src/assets/i18n/en.json (hunk 3 above)
Comment (asasin098676) [reply src/assets/i18n/en.json]: Resolved in 71246f23aebad5a25b7068cd1cd89704b5960a38

File: src/assets/i18n/en.json (hunk 4 above)
Comment (asasin098676) [reply src/assets/i18n/en.json]: Resolved in 71246f23aebad5a25b7068cd1cd89704b5960a38

File: src/assets/i18n/en.json (hunk 5 above)
Comment (asasin098676) [reply src/assets/i18n/en.json]: Resolved in 71246f23aebad5a25b7068cd1cd89704b5960a38

File: src/assets/i18n/uk.json (hunk 6 above)
Comment (asasin098676) [reply src/assets/i18n/uk.json]: Resolved in 71246f23aebad5a25b7068cd1cd89704b5960a38

**Review by aleks00onyshko — APPROVED** [review]

**Pull request merged**